test/evmc/evmc.h

```diff
@@ -44,7 +44,7 @@ enum
      *
      * @see @ref versioning
      */
-    EVMC_ABI_VERSION = 8
+    EVMC_ABI_VERSION = 9
```

Comment:
Are you sure you updated every relevant file in this directory?

Reply:
I actually did that in the second PR: https://github.com/ethereum/solidity/pull/11649/files

Will rebase this, once that's done.